lib/matplotlib/tests/test_dates.py

```diff
@@ -591,6 +596,25 @@ def test_rrulewrapper(attach_tz, get_tz):
     assert act == exp
 
 
+def test_rrulewrapper():
+    def attach_tz(dt, zi):
+        return dt.replace(tzinfo=zi)
+
+    _test_rrulewrapper(attach_tz, dateutil.tz.gettz)
+
+
+@pytest.mark.pytz
```

Comment:
What's this mark?

Reply:
All otherwise unspecified marks are used to register a test with the mark of that name. This indicates that the decorated test uses `pytz`, so that you can filter out tests that require pytz (or that *don't* require `pytz`).